### ISSUE TITLE
Train with other datasets collator/loader

### ISSUE THREAD

**Issue by chuplares** [OpenLMLab/LOMO#8]
Hello friend!

First of all, thanks for your amazing work!!

If I use another data collator/dataset loader, would I still be able to train using the LOMO trainer class?

**Comment by ayyyq** [comment]
Certainly! You have the flexibility to modify the dataset, data_collator, and compute_metrics according to your requirements. However, if your task does not involve multiple-choice question answering or generation, you'll also need to override the `eval_step` method within the LOMO trainer class. This allows you to customize the evaluation process to suit your specific task.
